code/modules/mob/living/carbon/human/human_defines.dm

```diff
@@ -73,3 +73,5 @@ var/global/default_martial_art = new/datum/martial_art
 
 	var/datum/body_accessory/body_accessory = null
 	var/tail // Name of tail image in species effects icon file.
+
+	var/list/splinted_limbs = list() //limbs we know are splinted

```

Comment:
Not to mention you're going to cause issues with this trying to update a limb that doesn't exist anymore or is dropped on the ground.